### ISSUE TITLE
data refresh question...

### ISSUE THREAD

**Issue by babloo80** [bvaughn/react-window#103]
Hello @bvaughn ,

I am pretty new to React, but I am already liking react-window (over react-virtualized for its simplicity). I do have a question on a handle/hook to refresh List/Grid's row(s) when the underlying data is updated? 
Please advice on some ideas / recommendations on how I could do this.

**Comment by TrySound** [comment]
You should pass your handler to itemData.
```js
const Item = ({ style, index, data }) => {
  return <div style={style} onClick={() => data.handleClick(index)}></div>
}

const List = () => {
  return <FixedSizeList ... itemData={{ handleClick: index => {} }}>{Item}</FixedSizeLIst>
}
```

**Comment by babloo80** [comment]
This is interesting. You recommend that I can tie the currently visible windowed to a state / reducer like hook and refresh it using (some sort of effect), when the remote data changes.
Is this correct?

**Comment by TrySound** [comment]
@babloo80 Nope, just pass any of your data to itemData. If they are immutable you will get correct result.

**Comment by babloo80** [comment]
@TrySound , you are super quick in the response. 
The data I use is remote and I use a custom implementation of `InfiniteLoader` like HOC, that buffers up-to n pages (where page is a size of single http fetch from the server).
Let me try to find a way of passing `itemData` function. I sense that, I may not be able to achieve this `immutable`-ly. Let me give it a try anyways?

**Comment by babloo80** [comment]
thanks for the help. Let me do as you have suggested @bvaughn
